lua/terrortown/entities/roles/ttt_role_base/shared.lua

```diff
@@ -99,6 +100,15 @@ function ROLE:IsBaseRole()
 	return self.baserole == nil
 end
 
+---
+-- Checks whether it is a special @{ROLE}.
+-- @note This just returns <code>false</code> if the role is Innocent!
+-- @return boolean Returns true if the player has a special role
+-- @realm shared
+function ROLE:IsSpecialRole()
+	return self.subrole ~= ROLE_INNOCENT
+end
```

Comment:
Somehow i thin this function needs to be justified, why do we need it at all? What is a special role and why would a role overwrite this anyways? I dont really see the usecase here

Reply:
I don't think that a role is to decide whether it is special or not that really depends on the viewer

Reply:
okay, I move it back to the player then